lib/plugins/aws/invokeLocal/index.js

```diff
@@ -312,60 +312,63 @@ class AwsInvokeLocal {
       this.serverless.cli.consoleLog(JSON.stringify(result, null, 4));
     }
 
-    const callback = (err, result) => {
-      if (!hasResponded) {
-        hasResponded = true;
-        if (err) {
-          handleError.call(this, err);
-        } else if (result) {
-          handleResult.call(this, result);
+    return new Promise((resolve) => {
```

Comment:
This is probably easier to review by turning off white space diffing: https://github.com/serverless/serverless/pull/5525/files?w=1